pkg/generate/ack/controller.go

```diff
@@ -51,6 +51,9 @@ var (
 		"ResourceExceptionCode": func(r *ackmodel.CRD, httpStatusCode int) string {
 			return r.ExceptionCode(httpStatusCode)
 		},
+		"ResourceExceptionMessagePrefixCheck": func(r *ackmodel.CRD, httpStatusCode int) string {
```

Comment:
Let's call the function `GoCodeResourceExceptionMessagePrefixCheck` since it outputs Go code (and the other functions that do so begin with `GoCode` :) )